src/components/search.tsx

```
@@ -93,26 +93,23 @@ export default function SearchBar() {
     }, [query, search])
 
     return (
-        <div className="relative h-full w-full">
+        <div className="relative">
             <div
-                className="hidden h-16 w-full flex-row items-center justify-between rounded-xl bg-background-secondary p-5 sm:flex"
+                className="bg-background-secondary hidden items-center justify-between gap-2 rounded-md p-3 sm:flex"
                 onClick={() => setOpen(true)}
             >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)